### DISCUSSION TITLE
Missing color.

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [ImageMagick/ImageMagick#2822] [Help] [answered]
Create color image and composite it with another
![background](https://user-images.githubusercontent.com/67605008/99080222-ad3c5a80-25c9-11eb-8b33-f95d6ad2d6bc.png)
![basic](https://user-images.githubusercontent.com/67605008/99080226-af061e00-25c9-11eb-87c4-2944eb5ae920.png)

Color missed

**Comment by jaka1-newbie** [comment]
IM7,  win7
basic.png  background.png -geometry +{-Variable.prCoord-}+86 -composite  basic.png

**Comment by jaka1-newbie** [comment]
Something with color bit...

**Comment by jaka1-newbie** [comment]
basic.png created by 
convert -size {-Variable.widthCover-}x2775 xc:{-Variable.backColor-} basic.png

**Comment by snibgo** [comment, marked as answer]
> basic.png background.png -geometry +{-Variable.prCoord-}+86 -composite basic.png

This isn't a complete command. Please always show your complete commands. I assume it starts with "magick".

I suppose your first input image is grayscale. To prevent the output from also being grayscale, insert "-set colorspace sRGB" like this:
```
magick basic.png -set colorspace sRGB background.png -geometry +{-Variable.prCoord-}+86 -composite basic.png
```

**Comment by jaka1-newbie** [comment]
You arу cool.
I post whole command. Work without magick
